zenodo/modules/records/serializers/to_marc21/rules.py

```diff
@@ -123,7 +123,12 @@ def reverse_files(dummy_self, key, value):
     }
 
 
-# @to_marc21.over('foobar', '^foobar$')
-# def to_leader_field(dummy_self, key, value):
-#     """Reverse - Files."""
-#     return value
+@to_marc21.over('041', '^language$')
```

Comment:
There seems to also be a `008/35-37` field [mentioned here](https://www.loc.gov/marc/bibliographic/bd041.html) and [here](https://www.loc.gov/marc/bibliographic/bd008a.html) (search for `008/35-37`) which is about language. I'm no MARC21 expert though...

Reply:
That is complementary, but it looks like we don't serialize the field 008 in the first place. If we did, we could put the the language code into it.